src/modules/request.js

```diff
@@ -128,7 +112,7 @@ export default class RequestManager {
    * @param {boolean} override whether the request can go through or not
    * @param {Boolean} sendOULFlag - true in case of a On User Login request
    */
-  saveAndFireRequest (url, override, sendOULFlag) {
+  saveAndFireRequest (url, override, sendOULFlag, evtName) {
```

Comment:
we are using this at many places and we are not passing evtName everywhere

Reply:
Have added evtName only for checking the event Name 'wzrk_wetch' and since the data received in url is encoded and compressed , can't extract it from url. 
This was done specifically to implement the new retry mechanism for fetchVariables.
https://wizrocket.atlassian.net/wiki/spaces/EN/pages/1803518283/Retry+mechanism